02.calendar/cal.js

```diff
@@ -0,0 +1,48 @@
+#! /usr/bin/env node
```

Comment:
`#!` の後ろのスペースは不要だと思います。

```suggestion
#!/usr/bin/env node
```

Reply:
不要なスペースを削除いたしました。